.github/workflows/draft-release.yml

```
@@ -0,0 +1,133 @@
+name: 'Draft Release'
+
+on:
+  workflow_call:
+    inputs:
+      version_strategy:
+        description: 'Version Strategy'
```

```suggestion
        description: 'Version strategy'
```

Can we elaborate on this description? Imagine someone who isn't as familiar with the release process.